Assets/MixedRealityToolkit.Examples/Experimental/HandMenuLayout/Prefabs/PressableButtonHoloLens2_HandMenu_Circular.prefab

```diff
@@ -0,0 +1,1622 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &1171651852186010226
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
```

Comment:
On CompressableCyldinersVisuals, there is a body, edge, and face gameobjects/meshes. 
Edge doesn't seem to really do anything that I can visually see when I toggle on/off
Body/Face also seem like they can be combined into one mesh with same material?

Reply:
Furthermore, how is this exactly different/needed compared to the prefabs "PressableButtonhololensCircular_nxn"?

Reply:
I've replaced the buttons in the circular hand menu with PressableButtonHololensCircular prefaband added the "body" and "face" geometry to improve the the highlight cage (they use the same material); also, removed "edge" geo (and its material).